app/components/auth/dummy/ui/DummyLoginButton.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import { NoUserAvatar } from "../../NoUserAvatar";
+import DummyLoginUI from "./DummyLoginUI";
+import styles from "../styles/DummyLoginButton.module.css";
+
+export default function DummyLoginButton() {
+  const [isOpen, setOpen] = useState(false);
```

Comment:
Use more explicit names. What is `open`ness supposed to represent here? suggestion: `isLoginUiOpen` never be afraid of over-explained variable names, most of us love them.